holoviews/element/chart.py

```diff
@@ -317,7 +317,7 @@ def stack(cls, areas):
             x, y = (area.dimension_values(i) for i in range(2))
             stacked[k] = area.clone((x, y+baseline, baseline), vdims=vdims,
                                     new_type=Area)
-            baseline += y
+            baseline = baseline+y
```

Comment:
nit: Would add spaces around the `+`.

Reply:
That said, would think `+=` works ok here. Was there an issue with it?

Reply:
The array ended up being shared between multiple stacked Area elements, which means they all incorrectly shared the same baseline. This was previously covered up by the fact that certain interfaces would copy the array anyway.

Reply:
Interesting. Guessing there is a test for this?